libs/domains/variables/feature/src/lib/create-update-variable-modal/create-update-variable-modal.tsx

```diff
@@ -221,6 +224,8 @@ export function CreateUpdateVariableModal(props: CreateUpdateVariableModalProps)
         'Variable are used at build/run time. Secrets are special variables, their value can only be accessed by the application.'
     )
 
+  console.log(type)
```

Comment:
to clean

Reply:
https://github.com/Qovery/console/pull/1576/commits/2ab42bfc3b848967b82c1ba75d68993d41a660aa